DAT-60126 ecomm app run mode edit calendar events

etrubek opened the pull request:
Added topFiltersRenderer function enable the option to add top components.
Also, added className to body rows

Chikel commented on app/src/components/base/Table.js:
```diff
@@ -57,7 +57,8 @@ const Table = props => {
     emptyRenderer,
-    onSearch
+    onSearch,
+    topFiltersRenderer
```
I think we better call it something like `tableHeaderRenderer` as `topFiltersRenderer` is specific for your use case.
But - looking at Abstract, I don't think we should render the filters as part of the table, these can just be on top of the table and you can remove the Table's search input.
Another renderer you'll probably need is `columnHeaderRenderer` for adding the sort icons.

etrubek replied:
1.Shay picked the name, topFiltersRenderer, I don't mind.
2. Regarding rendering the filters on top of the table, I suggested that option too, but we decided to go with topFiltersRenderer, again I don't mind.
3. I didn't use ColRenderer for the selectAll checkbox, don't have to use it here too.

Chikel replied:
@etrubek, ok then. I'm still not sure how you plan on adding the sorting icons to the column headers but I trust your judgment 👍

crazypixel replied:
@etrubek @Chikel 
let's change the name `tableHeaderRenderer` is better.
I agree with aran the filters can be outside the table component.
for the desc / asc icons. yes you need to add a `columnHeaderRenderer`

etrubek commented:
@crazypixel, changed name and updated version,
Looks find without colRenerre:
![image](https://user-images.githubusercontent.com/45935003/83620839-91527880-a596-11ea-91e5-1007383ed421.png)
